examples/chart/teleport-kube-agent/templates/_config.tpl

```diff
@@ -128,6 +128,21 @@ discovery_service:
   enabled: false
 {{- end }}
 
+jamf_service:
+  {{- if contains "jamf" (.Values.roles | toString) }}
+  enabled: true
+  api_endpoint: {{ required "jamfApiEndpoint is required in chart values when jamf role is enabled, see README" .Values.jamfApiEndpoint }}
+    {{- if .Values.jamfUsername }}
```

Comment:
In the config, if both username and client ID are set, username takes precedence. In the secret it's the opposite. Both should have the same behaviour in case of conflict.

Reply:
`jamfUsername` and `jamfPassword` have been removed.